extensions/interactions/EndExploration/EndExploration.js

```diff
@@ -73,9 +74,15 @@ oppia.directive('oppiaInteractiveEndExploration', [
         if (!$scope.isInEditorPreviewMode) {
           authorRecommendationsPromise.then(function() {
             var explorationId = explorationContextService.getExplorationId();
+            var collectionSuffix = '';
+            if ($scope.collectionId) {
+              collectionSuffix = '?collection_id=' + $scope.collectionId;
```

Comment:
We could introduce something like a UrlBuilder (I think something similar to this exists in the Closure API. I'm not convinced it should be solved in this PR, though. I could add a TODO or file an issue to solve it later.